### DISCUSSION TITLE
Explain template tags and including dot underscore

### DISCUSSION THREAD

**Discussion by boardtc** [akosbalasko/yarle#504] [General]
is there somewhere that explains templates?

On my tests .enex conversions

{created-at-block}Created at: {created-at}._{end-created-at-block}
{source-url-block}Source URL: []({source-url})._{end-source-url-block}

results in 

Created at: 2020-11-19._
Last updated at: 2020-11-19._

What's the idea of the ._ in between the tag elements resulting in them being at the end like this?

**Comment by akosbalasko** [comment]
Hi @boardtc ,
It is discussed and handled in https://github.com/akosbalasko/yarle/issues/507
The templates are detailed here: https://github.com/akosbalasko/yarle/blob/master/Templates.md

Best regards,
akos
